### DISCUSSION TITLE
Inputdata

### DISCUSSION THREAD

**Discussion by 66mengyu** [gempy-project/gempy#831] [Q&A] [answered]
hello,when I plot the input data,only seeing the few data,I have already set the extent,so why?what the unit of input data ?(X,Y,Z is meter?)

**Comment by AlexanderJuestel** [comment, marked as answer]
Hello @66mengyu, 

you have opened quite a lot of issues in the last couple of weeks. For new users, there are different resources available (documentation page, YouTube videos) that explain how GemPy works.

Have you had a look at them? 

The input data has the unit that you provided yourself. We recommend using coordinates in a cartesian coordinate system (meter), so avoid longitude and latitude coordinates in degrees. 

If you plot the data using the map view and plot_2d, it only plots the data inside the extent. Maybe you have data outside the extent?

Please also provide more information in the future and not just a one-liner that we have to interpret first. 

Cheers
Alex

**Reply by 66mengyu** [reply]
I'll try again,I have just touched and do feel that many basic questions are not very understood, I have seen some youtube videos, but I am not very clear about the logic of this input data yet, thank you very much for your patient answer during this time.

**Reply by AlexanderJuestel** [reply]
Alright, so where do you have some open questions? :)

**Reply by 66mengyu** [reply]
There's not problem for the time being, the others are still learning, and if there are problems in the future, I will trouble you to help, thank you very much. :)

**Reply by 66mengyu** [reply]
Hello,excuse me,I have another question.How to assign property data to the created geometry？

**Reply by AlexanderJuestel** [reply]
Hello, you could assign values using `geo_model.add_surface_values([1.5,1.5, 1.75, 2.5,4.5], ['vp'])`.

**Reply by 66mengyu** [reply]
ok,thank you.
